src/commands/build.js

```diff
@@ -59,7 +60,7 @@ exports.handler = async ({
 
   const mode = production ? 'production' : 'development';
   // Use env variables if working on Webpack >=5.
-  const projectsList = projects.split(',').filter((item) => item.length > 0);
+  const projectsList = projects.split(',').filter((item) => item.length > 0).map((item) => item.split('@')[0]);
```

Comment:
The order of these is somewhat incorrect. Mapping last will cause a project target of only entry points to return an array with a single empty string. If we've not defined any projects we need to check this before filtering or it will assume `@entrypoint` is the project.

A simple swap of logic order works.

```suggestion
const projectsList = projects.split(',').map((item) => item.split('@')[0]).filter((item) => item.length > 0);
```

Reply:
Made this change in my latest, commit. Wanted to test locally.